CHANGELOG

```diff
@@ -1,6 +1,15 @@
 devel
 -----
 
+* Added feature: DisjointSmartGraphs
+
+  SmartGraphs have been extended to a new subtype, called **DisjointSmartGraphs**.
```

Comment:
is `DisjointSmartGra` the official spelling?

Reply:
Yes